src/containers/Detail.jsx

```diff
@@ -2,4 +2,35 @@
 import React, { useState, useEffect } from 'react';
 import { getDetail } from '../services/AirbenderDetails';
 
+const DetailsList = () => {
+
+  const [details, setDetails] = useState([]);
+
+  useEffect(() => {
```

Comment:
I think you're going to have to send which character you've clicked on through to the URL and then grab it in this component to use your fetch. Otherwise it'll never know which one you clicked on. We did something similar in pokedex. You can also use useParams in this component to grab the characterId.